srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 2 nodes, setting nnodes to 1
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
Traceback (most recent call last):
IndexError: list index out of range
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
Traceback (most recent call last):
    main()
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
  File "run_linearization_heuristic.py", line 99, in main
    main()
    delta = all_instances[instance_index][0]
Traceback (most recent call last):
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
  File "run_linearization_heuristic.py", line 99, in main
IndexError: list index out of range
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
    delta = all_instances[instance_index][0]
    main()
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
IndexError: list index out of range
  File "run_linearization_heuristic.py", line 99, in main
IndexError: list index out of range
Traceback (most recent call last):
    delta = all_instances[instance_index][0]
  File "run_linearization_heuristic.py", line 871, in <module>
IndexError: list index out of range
    main()
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 871, in <module>
    delta = all_instances[instance_index][0]
    main()
Traceback (most recent call last):
IndexError: list index out of range
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 871, in <module>
    delta = all_instances[instance_index][0]
Traceback (most recent call last):
    main()
IndexError: list index out of range
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 99, in main
    main()
    delta = all_instances[instance_index][0]
  File "run_linearization_heuristic.py", line 99, in main
IndexError: list index out of range
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
Traceback (most recent call last):
Traceback (most recent call last):
IndexError: list index out of range
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431461.7
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431461.24
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431461.9
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431461.29
srun: Terminating job step 9431461.0
srun: Terminating job step 9431461.14
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431461.26
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431461.10
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431461.8
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431461.28
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431461.12
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431461.27
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431461.11
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431461.21
srun: Terminating job step 9431461.25
srun: Terminating job step 9431461.19
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431461.13
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431461.1
srun: Terminating job step 9431461.15
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431461.2
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431461.3
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431461.4
srun: Terminating job step 9431461.6
srun: Terminating job step 9431461.5
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431461.20
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431461.23
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431461.17
srun: Terminating job step 9431461.18
srun: Terminating job step 9431461.22
srun: Terminating job step 9431461.16
